omegatrade-app/backend/app/models/simulation.model.js

```diff
@@ -0,0 +1,62 @@
+'user strict';
+const database = require('./../config/database.js');
+const { v4: uuidv4 } = require('uuid');
+const Simulation = function () { };
+
+Simulation.getAll = async function () {
+    const [result] = await database.run({
+        sql: `SELECT sml.sId as sId,sml.companyId as companyId, sml.status as status, cy.companyName as companyName, cy.companyShortCode as companyShortCode FROM simulations sml 
+              LEFT JOIN companies cy ON sml.companyId = cy.companyId 
+              ORDER BY sml.createdAt DESC
+              LIMIT 3`,
```

Comment:
The name of the function in combination with this is slightly deceptive: It does not return all simulations, only the last three, so I think that either the name of the function or this `LIMIT` clause should be changed.

Reply:
yes Agreed, In frontend portal we are allowing only three simulations,so Limit clause in the above query removed.